t/t0091-sq-quote-buf-pretty.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+
+test_description='Testing the sq_quote_buf_pretty method in quote.c'
```

Comment:
This makes me think that the test file should be named for the "quote" file. Say, `t0091-quote.sh`. Yes, you only include tests for that method, but if we want to come in and carefully test quoting logic, this is where it would go.